spec/services/rapid_ready_for_decision/hypertension_upload_manager_spec.rb

```diff
@@ -26,13 +26,24 @@
     )
   end
 
+  let(:time_freeze_time) { '10-10-2021' }
+
+  before do
+    Timecop.freeze(time_freeze_time)
+  end
+
+  after do
+    Timecop.return
+  end
+
   describe '#add_upload(confirmation_code)' do
     context 'success' do
       it 'appends the new upload and saves the expected JSON' do
         RapidReadyForDecision::HypertensionUploadManager.new(form526_submission).add_upload('fake_confirmation_code')
         parsed_json = JSON.parse(form526_submission.form_json)['form526_uploads']
+
         expect(parsed_json).to match original_form_json_uploads + [
-          { 'name' => 'VAMC_Hypertension_Rapid_Decision_Evidence.pdf',
+          { 'name' => 'VAMC_Hypertension_Rapid_Decision_Evidence--10/10/2020-10/10/2021.pdf',
```

Comment:
Let's get rid of the slashes and make it sortable:
```suggestion
          { 'name' => 'VAMC_Hypertension_Rapid_Decision_Evidence--20201010-20211010.pdf',
```
or
```ruby
          { 'name' => 'VAMC_Hypertension_Rapid_Decision_Evidence--2020-10-10--2021-10-10.pdf',
```
Also, do we need to include both the start and end dates?

Reply:
> Let's get rid of the slashes and make it sortable:
> 
> or
> 
> ```ruby
>           { 'name' => 'VAMC_Hypertension_Rapid_Decision_Evidence--2020-10-10--2021-10-10.pdf',
> ```
> 
> Also, do we need to include both the start and end dates?

It's what the ticket asks for as far as representing the span of time the search was made. What is your concern?

Reply:
Concerns:
1. Slashes in filenames can cause problems in old OSes and can cause bugs if not treated properly.
2. Having both dates is a possible source of confusion.
3. Opinion: making the date string sortable is generally a good idea, so `YYYY-MM-DD`.

Reply:
@yoomlam Oh, I see — I agree on all those points. I was asking about your concern specifically over having both dates.

Reply:
Following up with design to make sure!